db_upgrade_script/mosip_esignet/sql/1.8.0_to_2.0.0_upgrade.sql

```
@@ -4,4 +4,6 @@
 
 \c mosip_esignet
 
-INSERT INTO esignet.KEY_POLICY_DEF(APP_ID,KEY_VALIDITY_DURATION,PRE_EXPIRE_DAYS,ACCESS_ALLOWED,IS_ACTIVE,CR_BY,CR_DTIMES) VALUES('BASE', 1095, 50, 'NA', true, 'mosipadmin', now());
\ No newline at end of file
+INSERT INTO esignet.KEY_POLICY_DEF(APP_ID,KEY_VALIDITY_DURATION,PRE_EXPIRE_DAYS,ACCESS_ALLOWED,IS_ACTIVE,CR_BY,CR_DTIMES) VALUES('BASE', 1095, 50, 'NA', true, 'mosipadmin', now());
+DELETE FROM esignet.KEY_POLICY_DEF WHERE APP_ID='BINDING_SERVICE';
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)